server/src/main/java/com/hexacore/tayo/reservation/ReservationController.java

```diff
@@ -0,0 +1,61 @@
+package com.hexacore.tayo.reservation;
+
+import com.hexacore.tayo.common.ResponseDto;
+import com.hexacore.tayo.reservation.dto.CreateReservationDto;
+import com.hexacore.tayo.reservation.dto.CreateReservationRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    @PostMapping()
+    public ResponseEntity<ResponseDto> createReservation(
+            @ModelAttribute CreateReservationRequestDto createReservationRequestDto) {
+        CreateReservationDto createReservationDto = CreateReservationDto.builder()
+                .carId(createReservationRequestDto.getCarId())
+                .rentDate(createReservationRequestDto.getRentDate())
+                .returnDate(createReservationRequestDto.getReturnDate())
+                .build();
+
+        ResponseDto responseDto = reservationService.createReservation(createReservationDto);
+        return ResponseEntity
+                .status(responseDto.getCode())
+                .body(responseDto);
+    }
```

Comment:
서비스에 넘길 정보가 요청시에 받은 데이터와 정확히 일치한다면 굳이 똑같은 클래스를 만들 필요가 없을 것 같아요.
그리고 응답 공통 형식은 다시 바뀔 예정이라 그거대로 적용하면 좋을 것 같아요!

```suggestion
            @ModelAttribute CreateReservationRequestDto createReservationRequestDto) {
        ResponseDto responseDto = reservationService.createReservation(createReservationRequestDto);
        
        return new ResponseEntity<>(responseDto, HttpStatusCode.valueOf(responseDto.getCode()));
    }
```